core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java

```diff
@@ -119,8 +119,15 @@ synchronized private static Configuration getXmlConfig() {
 
   @Override
   public String get(Property property) {
-    String key = property.getKey();
+    return get(property, true);
+  }
 
+  public String get(Property property, boolean useDefault) {
```

Comment:
Instead of putting this functionality into SiteConfig did you consider creating another config layer bettween DefaultConfig and SiteConfig?

Reply:
I want anything set on the command line (i.e CliConfig) to override SiteConfg so I don't know a layer between SiteConfig & DefaultConfig makes sense

Reply:
oh right, what about a layer after site config?

Reply:
A layer before SiteConfig could require a lot of code change but let me know if you have any ideas.

Reply:
I wouldn't bother creating a whole new layer on top of SiteConfiguration. SiteConfiguration is poorly named. It's really "Local Configuration" or "Startup Configuration". In any case, too much mucking with the configuration hierarchy could conflict with other efforts to clean up TableConfiguration and such.